libmamba/src/api/list.cpp

```diff
@@ -208,6 +231,7 @@ namespace mamba
         detail::list_options options;
         options.full_name = config.at("full_name").value<bool>();
         options.no_pip = config.at("no_pip").value<bool>();
+	    options.reverse =config.at("reverse").value<bool>();
```

Comment:
Indentation issue here but running `pre-commit run --all` in your source folder should fix it (I guess the linter is failing in the CI as well?)